skate-plugin/src/test/kotlin/com/slack/sgp/intellij/ChangeLogParserTest.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij
+
+import com.google.common.truth.Truth.assertThat
+import java.time.LocalDate
+import org.junit.Test
+
+class ChangeLogParserTest {
```

Comment:
Tests look mostly good! There's one missing though - a case where there is a matching entry that's not the first (i.e. we should take just the newer entries)

Reply:
Here is the commit for it: 3e57dfe91401dc63fac85654c3eb7e917c45b0d1